Fix nullable reference types warning

When building, the following warning is produced:

```
Languages/Python/Conversions.cs(29,74): warning CS8632: The annotation for nullable reference types should only be used in code within a '#nullable' annotations context. 
```

This warning should be addressed.

--- a/Freshli/Languages/Python/Conversions.cs
+++ b/Freshli/Languages/Python/Conversions.cs
@@ -1,4 +1,5 @@
-﻿using System;
+﻿#nullable enable
+using System;
 using System.Linq;
 
 namespace Freshli.Languages.Python {
@@ -16,7 +17,7 @@ namespace Freshli.Languages.Python {
       return value != null;
     }
 
-    public static string SafeToLower(string value) {
+    public static string? SafeToLower(string value) {
       if (!string.IsNullOrWhiteSpace(value)) {
         return value.ToLower();
       }
@@ -32,7 +33,7 @@ namespace Freshli.Languages.Python {
       return new long[] {};
     }
 
-    public static string SafeExtractString(string value) {
+    public static string? SafeExtractString(string value) {
       if (!string.IsNullOrWhiteSpace(value)) {
         return value;
       }

